routes/articles.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const express = require('express');
+const router = express();
+const bodyParser = require('body-parser');
+const jsonParser = bodyParser.json();
+const articleRoute = require('../database/articles.js');
```

Comment:
Again, super minor but you don't need to specify `.js` extension in module `require` statements. Makes it look consistent by names like w/ the rest of your node modules.